flytekit/loggers.py

```diff
@@ -186,5 +187,9 @@ def get_level_from_cli_verbosity(verbosity: int) -> int:
         return logging.DEBUG
 
 
+def is_display_progress_enabled() -> bool:
+    return os.getenv(FLYTEKIT_DISPLAY_PROGRESS_ENV_VAR, False)
```

Comment:
use https://github.com/flyteorg/flytekit/blob/5607b0d6d14b99dbc3442aaff40f98da02a4fabd/flytekit/core/utils.py#L402 instead:
```suggestion
    return str2bool(os.getenv(FLYTEKIT_DISPLAY_PROGRESS_ENV_VAR)
```

Reply:
I tried using `str2bool` here but we end up with a circular import here. Do you have a preference of fixing this?